boards/STM32F407ZET6-STM32_F4XX.md

```diff
@@ -0,0 +1,452 @@
+        <td><a href="http://www.advanced-monolithic.com/">Advanced Monolithic Systems Inc.</a></td>
+    </tr>
+    <tr>
+        <td>Regulator part number</td>
+        <td><a href="http://www.advanced-monolithic.com/pdf/ds1117.pdf">AMS1117</a></td>
+    </tr>
+    <tr>
+        <td>Regulator package</td>
+        <td>SOT223</td>
+    </tr>
+    <tr>
+        <td>Output</td>
+        <td>+3.3V @ 1000mA</td>
+    </tr>
+    <tr>
+        <td>Battery holder</td>
+        <td>Yes</td>
+    </tr>
+    <tr>
+        <td rowspan="6"><b>Connectivity</b></td>
+        <td>Headers</td>
+        <td>2x 2x30 male dupont (2.54mm)</td>
+    </tr>
+    <tr>
+        <td>Specific</td>
+        <td>TFT (2x16 male dupont (2.54mm))<br>NRF2401 (2x4 female dupont (2.54mm))</td>
+    </tr>
+    <tr>
+        <td>Debug</td>
+        <td>JTAG (20-pin IDC)</td>
+    </tr>
+    <tr>
+        <td>USB</td>
+        <td>Mini</td>
+    </tr>
+    <tr>
+	 <td>SD-card</td>
+	 <td>Micro SD-card socket</td>
+    </tr>
+    <tr>
+	<td>SDRAM</td>
+	<td>44-pin IS62WV51216BLL **Footprint only**</td>
+    </tr>
+    <tr>
+        <td rowspan="3>"><b>I/O</b></td>
+        <td>LEDs</td>
+        <td>Power LED (<b>D3</b>, +3.3V)<br>User LED (<b>D1</b>, <code>PF9</code>, active low)<br>User LED (<b>D2</b>, <code>PF10</code>, active low)</td>
+    </tr>
+    <tr>
+        <td>Buttons, switches and jumpers</td>
+        <td>Reset button (<b>REST</b>, <code>NRST</code>, active low)<br>User button (<b>K0</b>, <code>PE4</code>, active low)<br>User button (<b>K1</b>, <code>PE3</code>, active low)<br>User button (<b>WK_UP</b>, <code>PA0</code>, active high)<br>(BT0/BT1 jumpers are part of HEADER 2)</td>
+    </tr>
+    <tr>
+        <td>Other devices</td>
+        <td><a href="https://www.winbond.com/resource-files/w25q16jv%20spi%20revd%2008122016.pdf">Winbond W25Q16JV (16Mb FLASH)</a></td>
+    </tr>
+    <tr>
+        <td rowspan="3"><b>PCB</b></td>
+        <td>Colour</td>
+        <td>Blue</td>
+    </tr>
+    <tr>
+        <td>Size</td>
+        <td>95mm x 64mm</td>
+    </tr>
+    <tr>
+        <td>Mounting</td>
+        <td>4x mounting hole (M2)</td>
+    </tr>
+    <tr>
+        <td><b>Remarks</b></td>
+        <td colspan="2">
+            <ul>
+                <li><b>Warning:</b> The +5V pins on this board are directly connected to the +5V pin of the USB connector. There is no protection in place. Do not power this board through USB and an external power supply at the same time.</li>
+            </ul>
+        </td>
+    </tr>
+</table>
+
+## Resources
+
+ - [Schematic]({{ site.url }}/assets/pdf/boards/original-schematic-STM32F407ZET6-STM32_F4XX.pdf)
+
+## Pictures
+
+![STM32_F4XX board]({{ site.url }}/assets/img/boards/STM32F407ZET6-STM32_F4xx_1.jpg)
+
+![STM32_F4XX board Top view]({{ site.url }}/assets/img/boards/STM32F407ZET6-STM32_F4xx_2.jpg)
+
+![STM32_F4XX board Bottom view]({{ site.url }}/assets/img/boards/STM32F407ZET6-STM32_F4xx_3.jpg)
```

Comment:
Could you upload an image without the red text? I'd like to keep text out of the images, because this text can not be searched through by search engines and the ctrl+f of your browser. Also, it can not be automatically translated. Please add a reference to the footprint reference (U2) to the IS62WV51216BLL footprint "definition" in the table above.